Makefile

```diff
@@ -240,6 +245,9 @@ deploy-mainnet: build deploy
 
 preconfigure-mainnet: CONFIG_BASE_PATH=./devtools/config/mainnet/01
 preconfigure-mainnet:
+	# Validate RPCs
+	$(VALIDATE_RPCS) --config ./hardhat.config.ts
+
```

Comment:
Please note that this task failing will not prevent the rest of the configurations from continuing. This was done on purpose in the original task to avoid blocking users from proceeding if the task fails.